controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -145,7 +145,7 @@ def run_match(supervisor: Supervisor) -> None:
         robot.getField('customData').setSFString('start')
 
     # ... then un-pause the simulation, so they all start together
-    supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_REAL_TIME)
+    supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_RUN)
```

Comment:
possibly a controversial change? I'm worried teams might use normal `time` instead of `R.time` to measure timings, and thus if we run it at faster-than-realtime it might cause issues. We'd need to inform teams more clearly about the behavior of the simulation if we ship this.

Reply:
Actually, I take it back. It seems 0 teams used `time.sleep()` in the finals last year

Reply:
Also I've noticed that real-time is an approximation that swings between 0.9x and 1.1x so teams using time.sleep are going to be having issues anyway